style/crosswords.less

```diff
@@ -218,7 +157,6 @@
 
   .crossword-clue-label {
     display: inline-block;
-    width: @clue-label-width;
```

Comment:
The `width` value has been deleted. Please re-instate. It is used in  `dev/index.less`. The value of this variable may change in the future - we need the width for _multi-segment_ clues.

Reply:
My mistake - must have been a bad resolution to the merge conflict after #42. Reinstated as a CSS variable.